cfgov/cfgov/settings/base.py

```diff
@@ -113,6 +114,9 @@
 # Absolute path to the directory static files should be collected to.
 STATIC_ROOT = os.path.join(PROJECT_ROOT, 'static')
 
+MEDIA_ROOT = os.path.join(PROJECT_ROOT, 'media')
+MEDIA_URL = '/media/'
+
```

Comment:
Nah it just made sense at the time from a Wagtail perspective. On WP live they're under the directory `f`. It may make sense to change this to that.